stored_requests/backends/file_fetcher/fetcher.go

```diff
@@ -5,126 +5,133 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"path"
+	"path/filepath"
 	"strings"
 
 	"github.com/prebid/prebid-server/stored_requests"
 )
 
-// NewFileFetcher _immediately_ loads stored request data from local files.
-// These are stored in memory for low-latency reads.
-//
-// This expects each file in the directory to be named "{config_id}.json".
-// For example, when asked to fetch the request with ID == "23", it will return the data from "directory/23.json".
+// NewFileFetcher lazy-loads various kinds of objects from the given directory
+// Expected directory structure:
+// - stored_requests/{id}.json for stored requests
```

Comment:
Compatibility Check: are the directories unchanged in this PR?

Reply:
Directories are unchanged.

The directory structure documented here reflects the fact the original fetcher loads both requests and categories from the same root, the differentiation happens by which methods are called.

The original comment left out this information about the structure understood by the fetcher, which is not easy to discover. I can add more verbiage to indicate that the data sets depend on the data type.

Reply:
I think the comment is good as is. Just wanted to verify this is indeed the structure currently in place. As you said, it's not easy to discover. This should help a lot. Thank you.